### PR TITLE
CI-939 CI-982 CI-981 CI-968 CI-847 CI-829 CI-654 CI-684 CI-841 CI-904 CI-905 CI-906 CI-907 CI-929 CI-937 CI-623 CI-967 CI-966 CI-870 Account Funds tab

### PR DESCRIPTION
**Pull request by froberg**

### CODE REVIEW

File: src/SFA.DAS.Forecasting.Web/Orchestrators/Estimations/EstimationOrchestrator.cs [hunk 1]
```diff
@@ -84,5 +103,35 @@ public class EstimationOrchestrator : IEstimationOrchestrator
...
+                    EstimatedCost = projection.ModelledCosts.FundsOut,
+                    Balance = balance
+                };
+            });
+
+            return accountFumds.ToList();
+        }
+    }
+
+    public class AccountFunds
```
Comment (abroodie) [inline src/SFA.DAS.Forecasting.Web/Orchestrators/Estimations/EstimationOrchestrator.cs]: This class should be in a separate file in a relevant namespace.

File: src/SFA.DAS.Forecasting.Web/Orchestrators/Estimations/AccountFundsViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.Forecasting.Web.Orchestrators.Estimations
+{
+    public class AccountFundsViewModel
```
Comment (abroodie) [inline src/SFA.DAS.Forecasting.Web/Orchestrators/Estimations/AccountFundsViewModel.cs]: This class belongs in the ViewModels namespace

File: src/SFA.DAS.Forecasting.Web.UnitTests/OrchestratorTests/EstimationOrchestratorTests.cs [hunk 3]
```diff
@@ -138,5 +144,121 @@ public void Arrange()
...
+            first.EstimatedCost.Should().Be(0);
+            first.Balance.Should().Be(500);
+
+            second.ActualCost.Should().Be(0);
+            second.EstimatedCost.Should().Be(0);
+            second.Balance.Should().Be(1000);
+        }
+
+        [Test]
+        public async Task AccountFunds_should_not_have_any_costs_first_Month()
```
Comment (abroodie) [inline src/SFA.DAS.Forecasting.Web.UnitTests/OrchestratorTests/EstimationOrchestratorTests.cs]: AccountFunds for a Levy Estimation will contain the last months balance.

**Comment by froberg** [comment]
@mapaux 
Can you fix this conflicts please.

**Pull request merged**